apps/archive/common.py

```diff
@@ -428,6 +428,10 @@ def item_schema(extra=None):
                 'marked_for_legal': {
                     'type': 'boolean',
                     'default': False
+                },
+                'marked_archived_only': {
```

Comment:
Probably should but can't see a nice way, since we are only needing to add a flag to the schema within a already defined field. I'd have to pass the definition of all flags in extra_fields.

Can you see a better way?